### DISCUSSION TITLE
Menu badge is not displaying

### DISCUSSION THREAD

**Discussion by s-fog** [laravel/nova-issues#4571] [Q&A] [answered]
- Laravel Version: ^9
- Nova Version: 4.11.1
- PHP Version: 8.1
- Database Driver & Version: mysql
- Operating System and Version: ubuntu 20.04

### Description:

I have MenuItem `MenuItem::resource(PuppyDraftNovaResource::class),`
Displaying like this
![image](https://user-images.githubusercontent.com/10043516/178734871-0d28443f-53fb-4f10-8197-51b96460f7c8.png)

Adding badge, like that `MenuItem::resource(PuppyDraftNovaResource::class)->withBadge(Badge::make('some', 'info'))`
Displaying like this
![image](https://user-images.githubusercontent.com/10043516/178734871-0d28443f-53fb-4f10-8197-51b96460f7c8.png)

Nothing happened. Can you help to add badge please?

**Comment by juse-less** [comment, marked as answer]
@s-fog What namespace are you using for the `Badge`?
There's a `Badge` _`Field`_, but also a `Badge` that is meant for the `Menu`.

So crossing my fingers that your issue is as easy as replacing `\Laravel\Nova\Fields\Badge` with `\Laravel\Nova\Badge`.
```php
use Laravel\Nova\Badge;

MenuItem::resource(PuppyDraftNovaResource::class)->withBadge(Badge::make('some', 'info'))
```

**Reply by s-fog** [reply]
Thanks for helping. No i have correct namespace
![image](https://user-images.githubusercontent.com/10043516/178774576-3e74c44e-50e4-4215-8bec-33defb1c981c.png)

**Reply by juse-less** [reply]
Bummer.
Can you open up the developer tools/console in your browser and see if you have any errors?
Like an error about something missing, or so.

Another thing could be to try and republish the Nova assets, by running below Artisan command.
```
php artisan vendor:publish --tag=nova-assets --force
```

**Reply by s-fog** [reply]
There is no errors inside console, but republishing assets helps. Thanks!
